src/main/java/com/rockwell/basics/animals/reptile/Snake.java

```diff
@@ -0,0 +1,14 @@
+package com.rockwell.basics.animals.reptile;
+
+public class Snake extends Reptile {
+
+    @Override
+    public void crawl() {
```

Comment:
Let's think about methods unification. Each animal moves. Crocodile crawls, bird flies, etc... Generally, this a good initial approach to how model animal world.

Reply:
I will create method eat() in Animal class/interface.